src/ucp/core/ucp_worker.h

```diff
@@ -261,6 +268,8 @@ typedef struct ucp_worker {
     ucp_rkey_config_t                rkey_config[UCP_WORKER_MAX_RKEY_CONFIG];
 
     struct {
+        ucs_time_t                   timeout;             /* Time period between keepalive rounds */
```

Comment:
why need to keep it on worker if it will be the same for all of them?
Imo, better keep on context

Reply:
context may have "auto" value which may be replaced by specific value... replacing inplace is not good idea

Reply:
@brminich any other comments?

Reply:
no need for replacing inplace. Can just add to the context, not to the config

Reply:
okay